src/zjs_ocf_server.c

```diff
@@ -271,17 +268,21 @@ static jerry_value_t ocf_respond(const jerry_value_t function_val,
     return promise;
 }
 
-static jerry_value_t create_request(struct server_resource *resource, oc_method_t method, struct ocf_handler *handler)
+static jerry_value_t create_request(struct server_resource *resource,
+                                    oc_method_t method,
+                                    struct ocf_handler *handler)
 {
-
     handler->resp = create_response(resource, method);
+
     jerry_value_t object = jerry_create_object();
```

Comment:
This was intentional... if you do that in this particular case then you end up calling both jerry_release_value and jerry_acquire_object. So two function calls where none are needed.

So I think it makes sense to leave it off in this kind of case. But admittedly, there's some chance someone will come along later and return with an error and not free it.